frontend/src/AddMovie/AddNewMovie/AddNewMovieSearchResultConnector.js

```diff
@@ -13,14 +13,14 @@ function createMapStateToProps() {
     (state) => state.queue.details.items,
     (state) => state.tmdbId,
     (isExistingMovie, isExclusionMovie, dimensions, queueItems, tmdbId) => {
-      const firstQueueItem = queueItems.find((q) => q.tmdbId === tmdbId);
+      const firstQueueItem = queueItems.find((q) => q.tmdbId === tmdbId && tmdbId > 0);
 
       return {
         isExistingMovie,
         isExclusionMovie,
         isSmallScreen: dimensions.isSmallScreen,
-        queueStatus: firstQueueItem ? firstQueueItem.status : null,
-        queueState: firstQueueItem ? firstQueueItem.trackedDownloadState : null
+        queueStatus: typeof firstQueueItem === 'object' ? firstQueueItem.status : null,
```

Comment:
What's this part fixing? I would have thought it worked before and the tmdbId above was the issue.

Reply:
undefined still catches without this change